### PR TITLE
mesa => 23.1.1

### PR DESCRIPTION
**Pull request by satmandu**
Builds properly:
- [x] `x86_64`
- [x] `armv7l` 

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/satmandu/chromebrew.git CREW_TESTING_BRANCH=mesa23.1.1 CREW_TESTING=1 crew update
```

### CODE REVIEW

File: manifest/x86_64/m/mesa.filelist [hunk 1]
```diff
@@ -50,10 +50,6 @@
 /usr/local/lib64/vdpau/libvdpau_nouveau.so.1
 /usr/local/lib64/vdpau/libvdpau_nouveau.so.1.0
 /usr/local/lib64/vdpau/libvdpau_nouveau.so.1.0.0
-/usr/local/lib64/vdpau/libvdpau_r300.so
-/usr/local/lib64/vdpau/libvdpau_r300.so.1
-/usr/local/lib64/vdpau/libvdpau_r300.so.1.0
-/usr/local/lib64/vdpau/libvdpau_r300.so.1.0.0
```
[the remark this answers is not in the document]
Comment (satmandu) [reply manifest/x86_64/m/mesa.filelist:56]: Different set of drivers for different architectures...

**Review by uberhacker — APPROVED** [review]

**Review by ryugenxd — COMMENTED** [review]
🥳

**Comment by ZentarXD** [comment]
Ok

**Pull request merged**